src/styles/colors.tsx

```diff
@@ -36,6 +36,15 @@ export enum Colors {
   greenStrong = '#08CB7A',
   lightGreen = '#E6F7EC',
 
+  successDark = '#137211',
+  successLight = '#F1FDF1',
+
+  warningDark = '#9C6E00',
+  warningLight = '#FFF9EA',
+
+  errorDark = '#C93717',
+  errorLight = '#FBF2F0',
```

Comment:
just checking that you've aligned with Nitya on these colors and their semantics? e.g. errors will now have a dark and light variation in future designs and they will be those values? all good if you've done this, but otherwise i think it will be confusing to use these in the future (e.g. if we need a `errorMediumDark`, `errorMediumDarkerDark` 🙈) and would recommend to either use existing colors in the closest shade / using one off hard coded colors in the components where you need them / naming these like `gray1` etc. if we don't have a clear purpose for them.

Reply:
I took these names from Figma because it looks like they are part of some library designers are using 😅 
But I will check with Nitya if are they stable as a color system and can be reusable by name or not.